scripts/docker_files/docker_file_ci_ubuntu_20_04/DockerFile

```diff
@@ -77,9 +77,18 @@ RUN apt-get update -y && apt-get upgrade -y && \
     cd /tmp/ParMmg_4d272bb/build && git checkout 4d272bb3d1a51d839a5fdfaa3eef93f340cafda3 && \
```

Comment:
If you agree, the old version will be kept for some time to avoid failures of the CI on those branches that are not merged with the master.